server/groqAPI.py

```diff
@@ -0,0 +1,28 @@
+from flask import Flask, jsonify, request
+from flask_cors import CORS
+from datetime import datetime
+import json
+import logging 
+import os
+from dotenv import load_dotenv
+from groq import Groq
+
+load_dotenv()
+
+
+logging.basicConfig(level=logging.DEBUG)
+
+client = Groq(api_key="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",)
```

Comment:
Don't hardcode the API key in the source code, use the environment variable instead.